cypress/support/step_definition/courses/task_with_course/finishTaskFromRoomBoardSteps.spec.js

```diff
@@ -0,0 +1 @@
+//TBD
```

Comment:
Is it really necessary to have a file for each action? I mean it's just one button with a submenue on the page. I think they can be grouped.